README.md

```diff
@@ -1,10 +1,13 @@
-[![Build Status](https://travis-ci.org/lightblue-platform/lightblue-core.svg?branch=master)](https://travis-ci.org/lightblue-platform/lightblue-core) [![Coverage Status](https://coveralls.io/repos/lightblue-platform/lightblue-core/badge.png?branch=master)](https://coveralls.io/r/lightblue-platform/lightblue-core?branch=master) [![Stories in Ready](https://badge.waffle.io/lightblue-platform/lightblue.svg?label=ready&title=Ready)](http://waffle.io/lightblue-platform/lightblue)
```

Comment:
The waffle batch only shows issues in a "ready" status.  And I think it picks only for the primary repository, which is https://github.com/lightblue-platform/lightblue.  That means it always shows "Ready: 0". So, there is no value.  See http://docs.lightblue.io/